sgtm/dioscuri/README.md

```diff
@@ -0,0 +1,126 @@
+        and has fewer settings. Available settigs are automatically shown or 
+        hidden depending on your selection
+   - d. Add text prompts. You can add as many as you like. You can include data 
+        from other variables
+   - e. If using Gemini 1.5 Pro you can include file prompts. Upload files to 
+        cloud storage and get their Google Cloud file path which will start with
+        gs://. Ensure you select the correct file type. Any file types not listed
+        are not available. See [documentation](https://cloud.google.com/vertex-ai/generative-ai/docs/model-reference/gemini)
+        for specific requirements and limitations 
+   - f. Add a system instruction. This is a powerful statement to direct the 
+        response e.g. "Keep response as short and friendly as possible"
+   - g. Add a default value which will be returned if the tag fails. You can include
+        other Google Tag Manager variables in this field.
+   - h. Choose your safety settings. Be aware tighter settings may cause the response
+        to fail more often depending on your prompts.
+   - i. Update the generation configuation settings if required. These can be used
+        to tweak the response length and other elements. See below for more details
+        on what each setting can be used for
+   - j. Save
+7. Use the variable in a tag which has a live trigger and you will see data returned
+
+## Example Use Case
+
+You are a retail client who wants to get a personalised message for a user when
+they log into your site. You have already got user data  in Firestore and it is
+accessible in sGTM as variables using [Artemis](./sgtm/artemis/README.md).
+
+Add some text prompts. In this example we added the following:
+
+1. You are a virtual salesperson for a retail company which sells clothes. Write 
+a greeting for {{Firestore User Data - First name}} and suggest the kinds of products 
+they might want to look at today. It is {{Returning Customer - boolean}} that she is
+a returning customer and it is {{Is High Value User - boolean}} that she is a high value 
+customers. They live in {{Firestore User Data - Address - City}}, {{Firestore User Data - 
+Address - Country}}.
+2. Here is some information about {{Firestore User Data - First name}} in a stringified 
+JSON format which you should parse: {{Custome Data}}
+3. For high-value customers make sure to say how much you value them. For low value 
+customers encourage them to buy more
+
+Variables in braces are automatically replaced with relevant values based on customer
+data in real time. For example this is the customer data held for a customer, John,
+in Firestore
+
+![dioscuri_example_customer_data.png](./img/dioscuri_example_customer_data.png)
+
+When we run the tag you can see Gemini returns a personalised response which is 
+aware of his favourte category and colour:
+
+![dioscuri_example_response.png](./img/dioscuri_example_response.png)
+
+
+
+## Explanation of Settings
```

Comment:
It sounds like these have been pulled from a cloud document somewhere. Might be worth linking that for more info